Contentstack.Management.Core/Attributes/CSMJsonConverterAttribute.cs

```diff
@@ -5,8 +5,8 @@
 using System.Collections.Concurrent;
 using System.Runtime.CompilerServices;
 
-[assembly: InternalsVisibleTo("Contentstack.Management.Core.Unit.Tests")]
-[assembly: InternalsVisibleTo("Contentstack.Management.Core.Tests")]
+[assembly: InternalsVisibleTo("Contentstack.Management.Core.Unit.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100875b827dbdbc1ad6c58e621a0be924edcd15521351576d70f7133cacbe16828f7c20121e7241d02a6ace417ddf516969cac84ee388fbcf150afbdc0ce8838f58504df23d22d066bf8bbe2adc18da247752a5ad6016a84961508bc03f1b90953bea883f88d9a34aac9c05bfc62845c294ab7aa50e6a1c03f2b29164f8371d74ac")]
```

Comment:
@nadeem-cs Can you use the .env for sensitive information.

Reply:
@sunil-lakshman This is not sensitive info. This is public key.